<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataInspectorWindow</class>
 <widget class="QMainWindow" name="DataInspectorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>PCSX2 Data Inspector</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/icons/AppIcon64.png</normalon>
   </iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabs">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="watchTab">
       <attribute name="title">
        <string>Watch</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QTreeView" name="watchTreeView"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="watchControls">
          <item>
           <widget class="QPushButton" name="watchAddButton">
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="watchRemoveButton">
            <property name="text">
             <string>Remove</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="watchClearButton">
            <property name="text">
             <string>Clear</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="watchControlsSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="watchLoadButton">
            <property name="text">
             <string>Load</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="watchSaveButton">
            <property name="text">
             <string>Save</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="globalsTab">
       <attribute name="title">
        <string>Globals</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLineEdit" name="globalsFilter">
          <property name="placeholderText">
           <string>Filter</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTreeView" name="globalsTreeView"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="globalsControls">
          <item>
           <widget class="QCheckBox" name="sortBySection">
            <property name="text">
             <string>Sort by section</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="sortByTranslationUnit">
            <property name="text">
             <string>Sort by translation unit</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="globalsControlsSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="stackTab">
       <attribute name="title">
        <string>Stack</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTreeView" name="stackTreeView"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
